addons/core/functions/fnc_activeSwRadio.sqf

```diff
@@ -25,8 +25,12 @@ private _result = nil;
     true;
 } count (assignedItems TFAR_currentUnit);
 
-if (player != TFAR_currentUnit && !isNil "TFAR_remoteRadio" && TFAR_remoteRadio) then {
-    _result = TFAR_remoteRadioItem;
+if (player != TFAR_currentUnit) then {
```

Comment:
This is not how you add lazy evaluation.
`if (player != TFAR_currentUnit && {!isNil "TFAR_remoteRadio"} && {TFAR_remoteRadio}) then {`
Would've done the job. I prefer as little indentation as possible.
Also we use 4 space indentation. This fix looks like it's using 2 space indentation.